docs/application/dotnet/guides/security/dpm.md

```diff
@@ -0,0 +1,99 @@
+# Device Policy Management
+
+
+The Device Policy Management (DPM) framework supports enterprise applications by providing IT (Information Technology) administrator functions to create security-aware applications. These applications are useful in scenarios where IT administrators require rich control over employee devices.
+
+DPM framework consists of:
+
+- **Device policy client library**: This contains all the device administration functions that a client application can call. The device policy client library communicates with the device policy manager using a built-in remote method invocation (RMI) engine.
+- **Device policy manager**: This manages all the device policies. It also provides interfaces for the device policy client library.
+
+The main features of the [Tizen.Security.DevicePolicyManager](https://samsung.github.io/TizenFX/master/api/Tizen.Security.DevicePolicyManager.html) namespace are:
+
+- Managing policies
+
+  You can [track the state between the device admin client and the device policy manager](#client_application) using the `DevicePolicyManager` class provided by [Tizen.Security.DevicePolicyManager](https://samsung.github.io/TizenFX/master/api/Tizen.Security.DevicePolicyManager.html).
+
+- Checking restrictions
+
+  You can [check the restriction states of the device](#client_application), such as camera, microphone, Wi-Fi, Bluetooth, and USB, using the properties of the policy classes. [Tizen.Security.DevicePolicyManager](https://samsung.github.io/TizenFX/master/api/Tizen.Security.DevicePolicyManager.html) provides these policy classes.
+
+The following figure illustrates the DPM framework process:
+
+**Figure: DPM framework process**
+
+![DPM framework process](./media/dpm-framework.png)
+
+## Prerequisites
+
+To use the methods and properties of [Tizen.Security.DevicePolicyManager](https://samsung.github.io/TizenFX/master/api/Tizen.Security.DevicePolicyManager.html), include the following in your application:
```

Comment:
@sabeena21 how about "include it in your application" as it is adding Tizen.Security.DevicePolicyManager namespace?

In other dotnet guide pages, we can see 
 - "To use the methods and properties of the Tizen.XXX.XXX namespace, include it in your application:", or 
 - "To use the methods and properties of the Tizen.XXX.XXX namespace, include Tizen.XXX namespace in your application:" here and there.

I think changing would helps us to get consistency.

Reply:
I changed the sentence to "include it in your application".

Reply:
@mijin85cho 
From the consistency point, we can change it.